src/util/measurementUncertaintyTool.js

```diff
@@ -0,0 +1,66 @@
+import Decimal from 'decimal.js';
+
+// Run node version with 14.18.3
+/**
+ * Custom measurement uncertainty rounding rules.
+ * @param  {number} inputValue            measured value from annotation
+ * @param  {number} uncertaintyValue      uncertainty from pixel diagonal
+ */
+
+/**
+ * Rounding
+ */
+
+function roundValue(inputValue, uncertaintyValue) {
+  const value = new Decimal(inputValue);
+  let i = 0;
+
+  // Check if uncertainty is smaller than 1
+
+  if (uncertaintyValue < 1) {
+    const belowOne = uncertaintyValue.toString().split('.')[1];
```

Comment:
Would this be more clear?
```suggestion
    const decimalPart = uncertaintyValue.toString().split('.')[1];
```

Reply:
fractionalPart could also be considered

Reply:
Is there no validation needed here? I.e. making sure that uncertaintyValue has a '.'
This same split is repeated lower in the code, so I suggest
```suggestion
    const [wholePart, decimalPart] = uncertaintyValue.toString().split('.');
```